test/acceptance/behave/requirements.txt

```diff
@@ -6,4 +6,4 @@ requests==2.7.0
 xmltodict==0.9.2
 fabric==1.10.2
 pymongo==3.0.3
-git+https://visitor:visitor@pdihub.hi.inet/fiware/fiware-iotqaUtils@develop
\ No newline at end of file
+git+https://github.com/telefonicaid/iot-qa-tools.git
```

Comment:
It is not a good idea in general to depend on a "moving picture". I mean, differently that happen with a fixed version number (which always gets the same bunch of software), a git repository changes every day, as new commits are added. If iot-qa-tools is not formally using versions, at least a githash should be used to ensure that the dependency is on a particular and inmutable snapshoot of the code (although I don't know exactly how would be the syntax in requieremnts.txt for that).